src/components/MyForms/index.js

```diff
@@ -31,9 +54,9 @@ const MyForms = () => {
       .sort((a, b) => {
         return new Date(b.createdAt) - new Date(a.createdAt);
       });
-    console.log("Table Forms", tableForms);
-    setTableForms(tableForms);
-  }, []);
+    setFormDrafts(drafts);
```

Comment:
can have a unified state for forms
```js
form: {
  status: 'draft'|'published',
  ...other_fields
}
```

Reply:
Can be ignored

Can be picked yup during refactor

Reply:
agreed, also `tableForms` doesn't even make sense, also during the refactor we can store forms on backend/nostr, not sure localstorage is a sustainable approach, what do you think? or we can give a giant red warning to the users, that these can be deleted and should be backed up?